### ISSUE TITLE
Very important notice for 2.1.0

### ISSUE THREAD

**Issue by NOSCOPEdev** [undreamai/LLMUnity#203]
### Describe the bug

**If you are using version 2.1.0, please select the target architecture as ARM64 (it's ARMv7 by default). Otherwise, this makes the build file on the phone unusable(unable to download and load models). I can't say this for everyone, but this was the case for me. I hope people with the same problems will see this issue. I figured this out since other AI projects I was using before had similar problems with the target architecture. I hope this helps!**

### Steps to reproduce

![Notice](https://github.com/user-attachments/assets/bb6dea13-866f-4776-8eac-a9a685ab32b4)

### LLMUnity version

2.1.0

### Operating System

None

**Comment by amakropoulos** [comment]
Yes, that's correct. 2.1.0 is still work in progress, I haven't released it yet :slightly_smiling_face: .
I'll certainly add this to the Readme :+1:

**Comment by amakropoulos** [comment]
I'll even use your image, thanks :slightly_smiling_face: !
